### DISCUSSION TITLE
Do I need A and CNAME records when connection a GoDaddy domain?

### DISCUSSION THREAD

**Discussion by yaronlevi** [vercel/vercel#3971] [General] [answered]
Hi (-:

I am trying to connect a domain from GoDaddy to Zeit. 
Going by [zeit's docs](https://zeit.co/guides/setup-godaddy-domain-now) on connectiong a domain to GoDaddy, you need to add the Nameservers in the Godaddy DNS panel.

This is how the GoDaddy panel looks now:

<img width="1319" alt="Screen Shot 2020-03-25 at 19 03 26" src="https://user-images.githubusercontent.com/5398977/77634344-768dd300-6f59-11ea-9dc3-c966443f97ab.png">

So by the docs, I need to replace the two rows with "NS", with rows provided by Zeit.
But my question is about the first two rows. The one with "A" and the one with "CNAME".
Should I :
remove them?
change their value with some value supplied by Zeit?

Thank you for any help on the issue 🙏

**Comment by paulogdm** [comment, marked as answer]
Hello @yaronlevi ,

A few points to consider:

- ZEIT Now does not offer an A record for your domain. Do not use A records in our platform or you can cause downtime in your site. In the best-case scenario, the site will work but CDN will be disabled.
- The "NameServers" needs to be changed in the "authority" section. If you do change the NameServers in the dashboard you have presented, the domain will break. Reach out to GoDaddy support for further information.
- We offer the CNAME/ALIAS `alias.zeit.co`. Use it if you don't want to change NameServers.

**Comment by yaronlevi** [comment]
Hi, I didn't quite understand the point you made on the "authority" section. The screenshot I've added is from the DNS setup of the domain. It's the same as the one mentioned in the official docs:
https://zeit.co/guides/setup-godaddy-domain-now

Regarding the CNAME record, so it should be
CNAME | www | alias.zeit.co ?

**Reply by paulogdm** [reply]
Correct for the CNAME part.
Regarding the NS, there is another section in your GoDaddy dashboard to change it.

**Comment by paulogdm** [comment]
This discussion will be outdated very soon. Please refer to our docs: https://vercel.com/docs/v2/introduction
